dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/DefaultSecurityService.java

```diff
@@ -252,7 +252,7 @@ public void prepareUserForInvite( User user )
             user.setUsername( username );
         }
 
-        String rawPassword = CodeGenerator.getRandomSecureToken( INVITED_USER_PASSWORD_LENGTH_BYTES );
+        String rawPassword = "Inv!t3#" + CodeGenerator.getRandomSecureToken( INVITED_USER_PASSWORD_LENGTH_BYTES );
```

Comment:
Is it too long now? Should it be something like this?

```suggestion
        String rawPassword = "Inv!t3#" + CodeGenerator.getRandomSecureToken( INVITED_USER_PASSWORD_LENGTH_BYTES - 7);
```

Reply:
Yeah, it was too long. Thanks Enrico.